dev-libs/beecrypt/beecrypt-4.2.1.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="A strong and fast cryptography toolkit"
+DESCRIPTION="BeeCrypt is an ongoing project to provide a strong and fast \
+cryptography toolkit. It includes entropy sources, random generators, block \
+ciphers, hash functions, message authentication codes, multiprecision integer \
+routines, and public key primitives."
+HOMEPAGE="http://beecrypt.sourceforge.net"
+COPYRIGHT="2009 Bob Deblier"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/beecrypt/beecrypt-4.2.1.tar.gz"
+CHECKSUM_SHA256="286f1f56080d1a6b1d024003a5fa2158f4ff82cae0c6829d3c476a4b5898c55d"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86 x86_64"
+
+PROVIDES="
+	beecrypt$secondaryArchSuffix = $portVersion
+	lib:libbeecrypt$secondaryArchSuffix = 0.0.0 compat >= 0
```

Comment:
I might be doing something wrong, but for me it only yields `.PackageInfo` and no other file. I think this library might not provide a dynamic library to link against.